### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by m1kola**
[Rendered version](https://github.com/mamicart/mamicart-backend/blob/m1kola-patch-1/README.md).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# mamicart-backend
+
+**Note:** It's educational project. We will help each other to learn something new and how to work in team.
```
Comment (jabbson) [inline README.md]: Indefinite article "a" seems to be oddly missing, should be "work in a team".

Comment (m1kola) [inline README.md]: Fixed. Thanks!

**Comment by jabbson** [comment]
Although README most definitely will require some improvements in the future, I think it looks pretty good for this stage of the project.

**Comment by m1kola** [comment]
@jabbson Agree. At least we need to add instructions on how to setup the project. I've create a separate issue for it (#7).

**Pull request merged**